<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAdmin</class>
 <widget class="QFrame" name="FormAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QScrollArea" name="areaOption">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>481</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>129</width>
      <height>479</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_ajout">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_editer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>Editer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_quitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>591</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>419</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Page 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>419</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Page 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_quitter</sender>
   <signal>clicked()</signal>
   <receiver>FormAdmin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
